src/hooks/notifications/useToggleNotifications.ts

```diff
@@ -0,0 +1,33 @@
+import { useSetSetting, useSetting } from '@recoil/settings';
+import usePermissions from 'hooks/permissions/usePermissions';
+import { AppPermissions, AppPermissionStatus } from 'types/permissions';
+import React from 'react';
+
+/**
+ * Hooks that provides a function to enable or disable the reception of
+ * notifications and if the notification reception is enabled.
+ */
+const useToggleNotifications = () => {
```

Comment:
Is this needed? Can't the user simply disable the notifications from their OS (i.e. inside Android's system options)? This way they can disable them based on the notification channel as well.

Reply:
I added this because we need to request permissions for posting notifications. This is necessary for both iOS and Android starting from version 13. With this addition, when users enable notifications from the settings screen, the application will prompt for permission. In the future, this function could also be adapted for use, for example, in a popup that asks users if they want to enable notifications.

Reply:
I suggest temporarily disabling this feature for now. To enable notifications, users need a quick onboarding process where they'll be prompted to sign a transaction. This step ensures they can link their own token to their address. Given this requirement, it makes sense to deactivate the button in the settings temporarily. We can plan for a more improved onboarding experience in the future.

Reply:
@RiccardoM I've hidden the notifications toggle from the settings screen in the release version.